5.15.10-alpine/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM openjdk:8-jre-alpine
+
+ENV ACTIVEMQ_VERSION 5.15.10
+ENV ACTIVEMQ apache-activemq-$ACTIVEMQ_VERSION
+ENV ACTIVEMQ_TCP=61616 ACTIVEMQ_AMQP=5672 ACTIVEMQ_STOMP=61613 ACTIVEMQ_MQTT=1883 ACTIVEMQ_WS=61614 ACTIVEMQ_UI=8161
+ENV SHA512_VAL=35cae4258e38e47f9f81e785f547afc457fc331d2177bfc2391277ce24123be1196f10c670b61e30b43b7ab0db0628f3ff33f08660f235b7796d59ba922d444f
```

Comment:
you'll need to update the SHA to the correct one from https://archive.apache.org/dist/activemq/5.15.10/apache-activemq-5.15.10-bin.tar.gz.sha512